sdk/python/lib/pulumi/policy/__main__.py

```diff
@@ -50,7 +50,8 @@ def main():
                 project=os.environ["PULUMI_PROJECT"],
                 stack=os.environ["PULUMI_STACK"],
                 dry_run=os.environ["PULUMI_DRY_RUN"] == "true",
-                organization=os.environ["PULUMI_ORGANIZATION"],
+                # PULUMI_ORGANIZATION might not be set for filestate backends
+                organization=os.environ.get("PULUMI_ORGANIZATION", None),
```

Comment:
I think None is implied if a key does not exist.

Reply:
You've also been doing to much Go: https://docs.python.org/3/library/stdtypes.html#typesmapping

Reply:
I think Kyle meant that you don't have to explicitly specify the `None` to `get`. `None` is the default return value when a key isn't found.

```python
organization=os.environ.get("PULUMI_ORGANIZATION")
```

> get(key[, default])[¶](https://docs.python.org/3/library/stdtypes.html#dict.get)
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).

Which is different from the previous implementation using `d[key]`, which will raise `KeyError`.

> d[key]
> Return the item of d with key key. Raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError) if key is not in the map.